### PR TITLE
modified 0000-01-11-gaadevyshnav.mdv1.0

### PR DESCRIPTION
**Pull request by gaadevyshnav**
Added caption

### CODE REVIEW

**Comment by a-a-ron** [comment]
Hi, this looks great!

File: _posts/0000-01-11-gaadevyshnav.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-agh](https://cloud.githubusercontent.com/assets/16547949/25400895/02672eca-29c2-11e7-87a5-3dc3d3110c29.jpg)
 {: .center}
 
-Caption here
+Initial commit
```
Comment (a-a-ron) [inline _posts/0000-01-11-gaadevyshnav.md]: Nice caption

File: _posts/0000-01-11-gaadevyshnav.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-agh](https://cloud.githubusercontent.com/assets/16547949/25400895/02672eca-29c2-11e7-87a5-3dc3d3110c29.jpg)
 {: .center}
 
-Caption here
```
Comment (ssuryaextr) [inline _posts/0000-01-11-gaadevyshnav.md]: Nice!

**Review by a-a-ron — CHANGES_REQUESTED** [review]
This looks great, but let's update the title on line 3 to say something different.

**Review by ssuryaextr — APPROVED** [review]
Looks good.

**Review by a-a-ron — APPROVED** [review]

File: _posts/0000-01-11-gaadevyshnav.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Updated title!"
```
Comment (osuvankulov) [inline _posts/0000-01-11-gaadevyshnav.md:3]: looks good

**Pull request merged**